include/ttmlir/Dialect/TTNN/IR/TTNNOpsAttrs.td

```diff
@@ -105,4 +105,55 @@ def TTNN_MeshShapeAttr : TTNN_Attr<"MeshShape", "mesh_shape"> {
   let assemblyFormat = "custom<VargDimensionList>($y, $x)";
 }
 
+def TTNN_TensorConfig: TTNN_Attr<"TensorConfig", "tensor_config"> {
```

Comment:
Per @odjuricicTT's comment about naming.  FWIW TTNN has class called `class TensorLayout`.  Perhaps we should at least call it that?

Reply:
These are situations I wanted to avoid :D
<img width="778" alt="image" src="https://github.com/user-attachments/assets/c941a2e3-2a52-4cfd-8ecb-0e121aa71707">
Also layout gets mentioned during discussions frequently and I'm not sure which layout we are talking about :D
Anyway we can rename easily if we want to do in future.